### PR TITLE
Updated default Timeout value and error in example

### PR DESCRIPTION
**Pull request by RutujaShirali**
# PR Summary

The default -Timeout or the default Recursion Timeout value is set to 0x8 and not 0xF.

The example talks about setting the -RetryInterval to 3 however the output shows it being set to 15. I changed the example to show we are trying to set 15

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
@RutujaShirali 
Hello, 
We can merge this PR after you complete the **PR Checklist**.

image


Please @ mention me after it's complete.
Thank you.

**Comment by RutujaShirali** [comment]
@tiburd I have completed the PR checklist. Please let me know if you need anything else.

-Rutuja

**Pull request merged**
